themes/classic/templates/catalog/_partials/miniatures/product.tpl

```diff
@@ -23,9 +23,9 @@
  * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
  *}
 {block name='product_miniature_item'}
-<div itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
-  {if isset($position)}<meta itemprop="position" content="{$position}" />{/if}
-  <article class="product-miniature js-product-miniature" data-id-product="{$product.id_product}" data-id-product-attribute="{$product.id_product_attribute}" itemprop="item" itemscope itemtype="https://schema.org/Product">
+<div>
+  {if isset($position)}<meta content="{$position}" />{/if}
```

Comment:
The meta isn't relative to microdata ?

Reply:
yes do I remove the complete line 27 `{if isset($position)}<meta content="{$position}" />{/if}` ?

Reply:
Meta removed